Project.toml

```diff
@@ -0,0 +1,33 @@
+name = "JuMP"
+uuid = "4076af6c-e467-56ae-b986-b466b2749572"
+repo = "https://github.com/JuliaOpt/JuMP.jl.git"
+version = "0.18.0"
```

Comment:
I do not understand why the version is `0.18` instead of `0.19`?

Reply:
Ups :-P